backend/battling/tests/test_view.py

```diff
@@ -0,0 +1,85 @@
+from django.test import Client, TestCase
+
+from model_bakery import baker
+
+from battling.models import PokemonTeam
+from pokemon.models import Pokemon
+from users.models import User
+
+
+class ListBattlesTest(TestCase):
```

Comment:
When writing tests for things that return a collection, test:
  -  Zero
  - One
  - Few
  - A lot
  - Exception
  - [The diff ](https://vinta.atlassian.net/wiki/spaces/development/blog/2021/07/14/2024079365/Testing+the+diff)